lib/lazy_render/version.rb

```diff
@@ -1,3 +1,3 @@
 module LazyRender
-  VERSION = "0.1.3"
+  VERSION = "0.2.0"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)